jerry-core/jmem/jmem-heap.c

```diff
@@ -288,6 +288,7 @@ jmem_heap_gc_and_alloc_block (const size_t size, /**< required memory size */
 
   void *data_space_p = jmem_heap_alloc (size);
 
+  /* cppcheck-suppress memleak */
   while (JERRY_UNLIKELY (data_space_p == NULL) && JERRY_LIKELY (pressure < max_pressure))
   {
```

Comment:
It is a nonsense, it can't be a memleak, data_space_p gets new value only if it is NULL.